nessie/lib/queries.py

```diff
@@ -112,10 +112,22 @@ def undergrads_schema():
 
 
 def get_all_student_ids():
-    sql = f"""SELECT sid FROM {asc_schema()}.students
-        UNION SELECT sid FROM {coe_schema()}.students
-        UNION SELECT sid FROM {undergrads_schema()}.students
-        UNION SELECT sid FROM {advisee_schema()}.non_current_students"""
+    if feature_flag_edl():
```

Comment:
Looks good to me, @johncrossman! If we want to try the granular flagging I introduced earlier, this line would presumably be `if app.config['FEATURE_FLAG_EDL_SIS_VIEWS']"` - not that I'm wedded to that way of doing it if there's a better one.

Reply:
Thanks @pauline2k  Yes, let's go granular. Amendment on the way.